docs/extend/plugins_volume.md

```diff
@@ -140,15 +162,15 @@ at the first mount request and deprovision at the last corresponding unmount req
 **Response**:
 ```json
 {
-    "Mountpoint": "/path/to/directory/on/host",
+    "Mountpoint": "/path/to/directory/inside/plugin",
```

Comment:
All the endpoints are applicable in both v1 and v2. However a few of them have slightly different behavior in v2, compared to v1.